packages/neuron-ui/src/locales/zh.json

```diff
@@ -631,6 +631,7 @@
       "user-defined-asset-tooltip": "该资产为未识别资产，Neuron 暂时无法处理，请使用第三方应用操作该资产。",
       "locked-asset": "已锁定",
       "locked-asset-tooltip": "锁定参数为 {{epochs}} epochs, 预计解锁时间为 {{year}}年{{month}}月{{day}}日(锁定时间根据区块链实际运行情况会有一定的误差)。",
+      "withdraw-asset-tooltip": "预计解锁时间为 {{year}}年{{month}}月{{day}}日 {{hour}}时{{minute}}分(根据区块链实际运行情况会有一定的误差)。",
```

Comment:
Do we need space between number and Chinese?

Reply:
Spaces are not mandatory in the [style guide](https://github.com/cryptape/document-style-guide/blob/master/docs/contents/number.md)

Reply:
Good to know that there is a document style guide.